aspnetcore/blazor/webassembly-performance-best-practices.md

```diff
@@ -133,6 +133,10 @@ Blazor WebAssembly offers two additional versions of <xref:Microsoft.JSInterop.I
 dotnet publish -c Release
 ```
 
+### Lazy load assemblies
+
+Dynamically load assemblies at runtime when the assemblies are required by components. For more information, see <xref:blazor/webassembly-lazy-load-assemblies>.
```

Comment:
We've been trying to avoid using the term "dynamic" since it is a little overloaded with concepts in ASP.NET and elsewhere.

```suggestion
Load assemblies at runtime when the assemblies are required by a certain route. For more information, see <xref:blazor/webassembly-lazy-load-assemblies>.
```